plutus-script-utils/src/Plutus/Script/Utils/Typed.hs

```diff
@@ -180,14 +185,23 @@ class PV1.UnsafeFromData sc => IsScriptContext sc where
     --    where
     --       wrap = mkUntypedValidator
     -- @
+    --
+    -- For debugging purpose, it may be of interest to know that in the default implementation,
+    -- the parameters are decoded in their apparition order
```

Comment:
I don't think "in their apparition order" is used much in English. I'd say "in the order they appear".